subsys/mpsl/Kconfig

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "Nordic MPSL"
+
+config MPSL_SIGNAL_THREAD_PRIO
+	# Hidden option to set the priority of the MPSL signal thread
+	int
+	default 8
```

Comment:
Why 8?

Reply:
Just to use the same as ble_controller. See BLECTLR_PRIO

Reply:
If it _needs_ to match `BLECTRL_PRIO`, you'd better have it `default BLECTLR_PRIO` that way you are guaranteed that they are matching.

Reply:
Yes, but at some point it should rather be the other way around, i.e. that BLECTLR_PRIO is default MPSL_SIGNAL_THREAD_PRIO. But then it should be renamed I think. We are working on splitting MPSL out of ble_controller library now, so that MPSL and ble_controlelr will be linked together in NCS. So we should fix it in that context.